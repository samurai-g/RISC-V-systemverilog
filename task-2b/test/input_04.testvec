A
aae9a075
c569a4b9
b2b7013f
6dbd3f50
b1cf19fa
618b6caa
3cd343c3
97075c50
0060a472
b577aaa7
