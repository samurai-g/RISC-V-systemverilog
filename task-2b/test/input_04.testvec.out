97075c50
aae9a075
b1cf19fa
b2b7013f
b577aaa7
c569a4b9
0060a472
3cd343c3
618b6caa
6dbd3f50
